src/lucky/tags/specialty_tags.cr

```diff
@@ -87,7 +87,7 @@ module Lucky::SpecialtyTags
       options.each_with_index do |key, value, index|
         attrs << ", " if index > 0
         attrs << Wordsmith::Inflector.dasherize(key.to_s) << "="
```

Comment:
Maybe we should make all of the wordsmith methods take an optional IO so we can do `.dasherize(io, key)` :thinking: